portfolio/src/main/webapp/index.html

```diff
@@ -1,18 +1,110 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <meta charset="UTF-8">
-    <title>My Portfolio</title>
-    <link rel="stylesheet" href="style.css">
-    <script src="script.js"></script>
-  </head>
-  <body>
-    <div id="content">
-      <h1>My Portfolio</h1>
-      <p>This is my portfolio.</p>
-      <p>Click here to get a random message:</p>
-      <button onclick="addRandomGreeting()">Hello</button>
-      <div id="greeting-container"></div>
-    </div>
-  </body>
-</html>
+    <head>
+        <meta charset = "utf-8">
+        <meta name = "viewport" content = "width=device-width, initial-scale=1.0"> 
+        <!--above: takes care of device width-->
+        <meta http-equiv="X-UA-Compatible" content = "ie=edge">
+        <title>Alicia's Portfolio | Welcome</title>
+        <link rel = "stylesheet" href="style.css">
+    </head>
+    <!--Body-->
+    <body>
+        <!--Start Header-->
+        <header id = "top">
+            <div class = "container">
+                <!--class vs id-->
+                <!--id's are unique for something that occurs once-->
+                <!-- a class is for something that can be used more than once-->
+                
+                <!-- navigation-->
+                <nav>
+                    <ul> <!--unordered list-->
+                        <li class = "current"><a href="http://127.0.0.1:5501/portfolio.html">Home</a></li>
+                        <li><a href="UpdatedCSResume2020.pdf" download>Resume</a></li>
+                        <li><a href="https://www.linkedin.com/in/alicia-simon-4b4454176/" target="_blank">LinkedIn</a></li>
+                        <li><a href="https://blacklivesmatter.com" target="_blank">#BLM</a></li>
+                    </ul>
+                </nav>
+            </div>
+        </header>
+        <!--Showcase-->
+        <section id="showcase">
+            <div id="profile">
+                <!--profile image in circle-->
+                <img src="images/profilepic.png">
+            </div>
+            <div id="intro">
+                <h1 id="name">Hi, I'm Alicia</h1>
+            </div>
+        </section>
+        <section id="AboutMe">
```

Comment:
Lets follow a convention to have a consistent naming for ids.

For instance here you have "AboutMe", but in the next section you have it as "txt".

Generally people prefer underscore or hyphens to separate words like "about-me".

Reply:
Just made those changes 👍